src/System.CommandLine.Hosting/Directory.Build.props

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <UseAlphaVersion>true</UseAlphaVersion>
+    <Nullable>enable</Nullable>
```

Comment:
Please make this change in the `.csproj` file for consistency with the other projects.

Reply:
Updated